### DISCUSSION TITLE
How can I override the partials?

### DISCUSSION THREAD

**Discussion by banstola** [gethinode/hinode#914] [Help] [answered]
I am getting following error when trying to deploy my website at netlify that I moved to using hinode. 
```
3:22:12 PM: Error: add site dependencies: load resources: loading templates: "/opt/build/cache/hugo_cache/modules/filecache/modules/pkg/mod/github.com/gethinode/hinode@v0.23.12/layouts/partials/assets/download.html:1:1":
 parse failed: template: partials/assets/download.html:1: function "page" not defined
```
I would maybe do not need download.html and what is the easy way to override this partial.

Thank you in advance!

**Comment by banstola** [comment]
I have already tried adding empty content to `layouts/partials/assets/download.html` but that has not prevented the error.

**Comment by markdumay** [comment]
Which version of Hugo are you using on Netlify? The global [page function](https://gohugo.io/functions/global/page/) was introduced in v0.111.0. 

The Hinode repo contains an example netlify.toml that may be useful: https://github.com/gethinode/hinode/blob/main/netlify.toml

**Reply by banstola** [reply]
I have 
```
 % hugo version
hugo v0.125.3-474c4c02212cf97712c6fbf4159c68822ea6e078+extended darwin/arm64 BuildDate=2024-04-22T17:18:35Z VendorInfo=brew
```

**Reply by markdumay** [reply]
This is the **local** macOS binary, not the Hugo version on the Netlify host…

**Comment by markdumay** [comment]
You might also want to check the docs: https://gethinode.com/docs/getting-started/hosting-and-deployment/#host-on-netlify

**Comment by banstola** [comment, marked as answer]
The solution was to specify netlify config as below using netlify.toml file at the root of the project

```
[build.environment]
    DART_SASS_VERSION = "1.72.0"
    HUGO_VERSION = "0.124.0"
    HUGO_ENV = "production"
    HUGO_ENABLEGITINFO = "true"
    NODE_VERSION = "18.19.1"
    NPM_VERSION = "10.2.4"
```

@markdumay Thank you for helping with this.
